webapp/channels/src/components/new_channel_modal/new_channel_modal.tsx

```diff
@@ -301,6 +243,10 @@ const NewChannelModal = () => {
         </OverlayTrigger>
     );
 
+    const onChannelDisplayNameChanged = useCallback((newName: string) => {
```

Comment:
Not needed, you can directly use `setDisplayName`.

Reply:
Done.